### ISSUE TITLE
Feature Request: add a per-project default directory sorting option

### ISSUE THREAD

**Issue by StevenClontz** [sagemathinc/cocalc#5037]
It'd be nice if users could define a default sorting of files. (In particular, the first click I make each time I open CoCalc is to start files by filename.) Apologies if this is already possible and I didn't see how!

**Comment by williamstein** [comment]
Account prefs has a default sort order option.

**Comment by StevenClontz** [comment]
Thanks. For whatever it's worth, I tried to find this in project options. Making it appear in user options instead makes sense, but maybe I'm not the only user who wouldn't mind having user option toggles available on the project options page.

**Comment by williamstein** [comment]
There are several design decisions if we add this:
 - where would the setting be in the user interface?
 - what's the scope:
       - *all* users of the project (so stored in project settings)
       - just for a specific user of the project (so stored in account settings)
       - for one web browser (so stored in localStorage in the browser)
- how does it relate to the global default sort order in account preferences?  Does this override it?

Note that while a project is open there is also a sort order for a specific project that you can manually set *during a session* (it goes away when you refresh the browser).    This is set by just clicking a heading in the file listing. 

So one solution to all these problems might be to make the currently implemented "project and session specific" sort order instead persist between sessions.  That would be be the only change.  If you use a different browser any prefs disappear, and what you do doesn't impact other users.

**Comment by StevenClontz** [comment]
Yeah I've been doing the per-session solution, after incorrectly attempting to set the preference by project rather than user.

I don't think I personally have a use case where I'd actually want to set user preferences like file sorting or Vim keybindings on a project-by-project basis. But for whatever reason I forget these things are under Account, and have looked for them under Project Settings in the past. Maybe even a reminder that the Account settings exist ("Can't find what you're looking for?") would be a sufficient solution.
